src/coreComponents/physicsSolvers/multiphysics/LagrangianContactSolver.cpp

```diff
@@ -112,18 +112,16 @@ void LagrangianContactSolver::registerDataOnMesh( Group & meshBodies )
           setDescription( "An array that holds the rotation matrices on the fracture." ).
           reference().resizeDimension< 1, 2 >( 3, 3 );
 
-        subRegion.registerWrapper< array2d< real64 > >( viewKeyStruct::tractionString() ).
-          reference().resizeDimension< 1 >( 3 );
-        subRegion.registerWrapper< array2d< real64 > >( viewKeyStruct::tractionString() ).
-          setApplyDefaultValue( 0.0 ).
+        auto & rwtTraction = subRegion.registerWrapper< array2d< real64 > >( viewKeyStruct::tractionString() );
+        rwtTraction.reference().resizeDimension< 1 >( 3 );
+        rwtTraction.setApplyDefaultValue( 0.0 ).
           setPlotLevel( PlotLevel::LEVEL_0 ).
           setRegisteringObjects( this->getName()).
           setDescription( "An array that holds the tractions on the fracture." );
 
-        subRegion.registerWrapper< array2d< real64 > >( viewKeyStruct::deltaTractionString() ).
-          reference().resizeDimension< 1 >( 3 );
-        subRegion.registerWrapper< array2d< real64 > >( viewKeyStruct::deltaTractionString() ).
-          setApplyDefaultValue( 0.0 ).
+        auto & rwtDeltaTraction = subRegion.registerWrapper< array2d< real64 > >( viewKeyStruct::deltaTractionString() );
+        rwtDeltaTraction.reference().resizeDimension< 1 >( 3 );
```

Comment:
Why sometime do you put prefix `rwt` and sometimes `rw`?
Also, I assume this stands for `register wrapper`, but I do not get the meaning.

Otherwise, OK for me.